src/Compilers/CSharp/Test/Symbol/Symbols/Source/NullablePublicAPITests.cs

```diff
@@ -2184,5 +2184,49 @@ void assertAnnotation(SingleVariableDesignationSyntax variable, PublicNullableAn
                 Assert.Equal(expectedAnnotation, symbol.NullableAnnotation);
             }
         }
+
+        [Fact, WorkItem(37659, "https://github.com/dotnet/roslyn/issues/37659")]
+        public void InvalidCodeVar_GetsCorrectSymbol()
+        {
+            var source = @"
+public class C
+{
+    public void M(string s)
+    {
+        s. // no completion
+        var o = new object;
+    }
+}
+";
+
+            var comp1 = CreateCompilation(source, parseOptions: TestOptions.Regular8.WithFeature("run-nullable-analysis"));
```

Comment:
consider factoring most locals suffixed with '1' or '2' into 'verifySymbol' here.

Reply:
Agreed. If there only difference is `WithFeature(...)`, consider all other code into `verifySymbol`.

---
In reply to: [310769041](https://github.com/dotnet/roslyn/pull/37726#discussion_r310769041) [](ancestors = 310769041)